### ISSUE TITLE
[BUG] Can't upgrade altar

### ISSUE THREAD

**Issue by NightmareTwilight** [HellFirePvP/AstralSorcery#967]
When trying to recreate #959 to check if it was working now, or not, I discoverd this bug. 
When trying to upgrade the altar nothing happened, I checked if there was enough starlight, but everything seem ok. Holding right click, random altar particles spawned,but nothing else. I have the latest Astral Sorcery, Baubles and VanillaFix instaled.

**Comment by Doomgull** [comment]
959 is fixed. Confirmed. You don't need to "test" that, it's been tested considerably. If you see something similar, it's a different issue now.

I just went through using only Astral and its dependencies with the latest version, and manually created all tiers of altar from the Luminous Crafting Table onward. It takes longer than before to craft, as intended, but otherwise works correctly. Try the issue without VanillaFix installed, and report to them as needed.
